packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -124,30 +124,14 @@ enum MaterialTapTargetSize {
 /// for the subtree that appears below the new [Theme], or insert a widget
 /// that creates a new BuildContext, like [Builder].
 ///
-/// {@tool snippet}
-/// In this example, the [Container] widget uses [Theme.of] to retrieve the
-/// primary color from the theme's [colorScheme] to draw an amber square.
+/// {@tool dartpad --template=stateless_widget_scaffold}
+/// This sample shows how to display a [Container] widget that uses [Theme.of]
+/// to retrieve the primary color from the theme's [colorScheme] to draw an
+/// amber square.
```

Comment:
You might add a blank line after this make a new paragraph in the markdown (if it doesn't already).